server/WebAPI/DTO/PaginationRequestQueryDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebAPI.DTO
+{
+  public class PaginationRequestQueryDTO
+  {
+    [Required]
+    [Range(0, Int32.MaxValue)]
+    public int? PageNumber { get; set; }
+
+    [Required]
+    [Range(0, Int32.MaxValue)]
```

Comment:
does it really need a minimum value of 0?

Reply:
Why not ?

Reply:
does anyone really need 0 elements when querying?) it doesn't seem logical to me, because if we make a request, we want to get at least something